### DISCUSSION TITLE
Bookmarking or starring issues

### DISCUSSION THREAD

**Discussion by tapio-jakamo** [community/community#37608] [Repositories]
Is there a way to bookmark or star tickets? Just like there's a way to do that with repositories.

I tried to do a project on my profile, but that did not work for me since one can only add tickets from repos you personally own there. I could do a project on my organisation, but that would be spamming other people.

**Comment by tapio-jakamo** [comment]
This feature has been asked by someone else too (https://github.com/community/community/discussions/16331). Is it already implemented?

**Comment by ossanna16** [comment]
Hi there @tapio-jakamo and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
